.golangci.yml

```diff
@@ -60,3 +57,9 @@ linters-settings:
       - G402
       # G601: Implicit memory aliasing in for loop.
       - G601
+issues:
+  exclude:
+    - "SA1019: \"io/ioutil\" has been deprecated since Go 1.16"
+  exclude-rules:
+    - path: server/artifacts/artifact_server_test.go
+      text: "response body must be closed"
```

Comment:
I think this was a false positive but let me know if there is an easy way to fix.

Reply:
Is it flaky or consistent?

Reply:
consistent